index.js

```diff
@@ -1,52 +1,50 @@
-'use strict'
+/* eslint-disable global-require */
 
 // require new relic at the top only in production environment
-if (process.env.NODE_ENV === 'production') {
-  require('newrelic')
-}
+if (process.env.NODE_ENV === 'production') require('newrelic');
```

Comment:
Is writing single line functions good ? @rahulthampi

Reply:
if you have only one line of body and it less than max-line limit then it is fine

Reply:
Cool.